guru99/test_parameterization.py

```diff
@@ -0,0 +1,34 @@
+from selenium import webdriver
+import time
+from selenium.webdriver.common.by import By
+import pytest
+import openpyxl
+
+
+#configure workbook path
+wb = openpyxl.load_workbook("C:\\Users\\jagad\\Dropbox\\PC\\Desktop\\Test_data.xlsx")
```

Comment:
Where is the Test_data.xlsx.  Suggesting you to place the all test data files under the path  --> guru99\test_data\ and you need check-in the test data folder as well